src/lib/modules/ens/embarkjs.js

```diff
@@ -252,13 +264,24 @@ __embarkENS.registerSubDomain = function (name, address, callback) {
   if (!this.registration || !this.registration.rootDomain) {
     return callback('No rootDomain is declared in config/namesystem.js (register.rootDomain). Unable to register a subdomain until then.');
   }
-  if (!address || !web3.utils.isAddress(address)) {
+  if (!address || !Web3.utils.isAddress(address)) {
     return callback('You need to specify a valid address for the subdomain');
   }
 
   // Register function generated by the index
-  registerSubDomain(this.ens, this.registrar, this.resolver, web3.eth.defaultAccount, name, this.registration.rootDomain,
-    web3.utils.soliditySha3(address.toLowerCase().substr(2) + reverseAddrSuffix), address, console, EmbarkJS.Utils.secureSend, (err, result) => {
+  registerSubDomain(
+    EmbarkJS.Blockchain.blockchainConnector.getInstance(),
+    this.ens,
+    this.registrar,
+    this.resolver,
+    EmbarkJS.Blockchain.blockchainConnector.getDefaultAccount(),
+    name,
+    this.registration.rootDomain,
+    Web3.utils.soliditySha3(address.toLowerCase().substr(2) + reverseAddrSuffix),
+    address,
+    console,
+    EmbarkJS.Utils.secureSend,
+    (err, result) => {
```

Comment:
Again, putting params on multiple lines and replacing web3 by according functions